src/awesomeSezzle.js

```diff
@@ -1187,14 +1189,20 @@ class AwesomeSezzle {
     sezzleModal.addEventListener('click', function (event) {
       event.stopPropagation();
     });
+    ////closes modal when escape key is hit
```

Comment:
There are dryer ways to do this. Isolate this into its own function, then call it inside each `sezzleModal.addEventListener`. Also, you'll either need to create a more unique way to differentiate between the competitor modal and ours in this event (by passing in the variable), or you'll need to getElementsByClassName and do a for loop.